tests/moe/test_trtllm_gen_fused_moe.py

```
@@ -1862,6 +1862,32 @@ def routing_reference_sigmoid_renorm(
+def routing_reference_minimax2(
+    expert_logits, routing_bias, top_k, num_experts, padding
+):
+    """Sigmoid + Bias -> TopK -> ScaledSumNormalize routing reference (MiniMax2).
+    Bias affects expert selection but NOT the final weights.
+    Weights = sigmoid(logit) / (sum_of_selected_sigmoid + 1e-20).
+    """
+    sigmoid_scores = torch.sigmoid(expert_logits.float())
+    selection_scores = sigmoid_scores.clone()
+    if routing_bias is not None:
+        selection_scores = selection_scores + routing_bias.float()
+    topk_values, topk_idx = torch.topk(selection_scores, k=top_k, dim=-1)
```

Ruff (0.15.9)

[warning] Unpacked variable `topk_values` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)